frontend/desktop/desktop.index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en", dir ="ltr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple Website <!-- will later be replaced--></title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body class="desktop">
+    <style>
+        @import url('https://fonts.cdnfonts.com/css/lucida-console');
+    </style>
+
+    <div>
+        <p> Hi there! Welcome to our desktop site! This experience was designed
+         to be on a mobile website,so we ask that you move onto your mobile 
+         device so that you have the best experience possible. Here’s a QR code
+         so you don’t have to re-enter it!</p>
+    </div>
+
+    <div>
+        <!--will later be replaced with actual QR code-->
+        <img src="images/ExampleQRCode.png" alt="QR Code for our mobile site"> 
+    </div>
+
+    <script src="isMobile.js">
```

Comment:
the first page the user lands on when they visit our page is going to be the `frontend/index.html` page, so this is the page which contain the mobile logic check. so i would remove this script tag from here && add it to the `frontend/index.html` page instead.

this means the logic will also need to change a little bit, instead of re-routing them to the home page, `index.html`, if they are on mobile, we're going to want to re-route them to the desktop page `desktop/index.html` if they are **not** on mobile (otherwise, we'll leave them on the homepage)

```js
if (!isMobile) { // the "!" means "not"
  window.location = 'desktop/index.html'
}
```
*note: in order to conform to our coding style, all JS files need to have an empty line at the end, ur file was missing that, so make sure to add that in when u make this change as well*

Reply:
also, it might make sense to create a new directory in the `frontend` folder called `js` && then move the `isMobile.js` out of the `desktop` folder && into the new `frontend/js` folder (seeing as that it's going to be used in the main `index.html` now && not simply inside the desktop index)